### DISCUSSION TITLE
How enable/setup HTTP/2 protocol on my domain

### DISCUSSION THREAD

**Discussion by HectorLS** [vercel/vercel#6601] [Help]
Hello,
Analysing some of my sites with Lighthouse analytics, done with Next.js & deployed on Vercel. most of them complain about keep using HTTP/1.1 instead of the HTTP/2 for my static files on Vercel.

I have done a deep web research to just find this link https://vercel.com/docs/edge-network/overview#supported-protocols. where HTTP/2 has a mention that is a supported protocols.

**It's just that point. How can i enable/setup  HTTP/2 protocol on my domain ?**

Any help would be much appreciated, thanks in advance.

**Comment by leerob** [comment]
Could you share a link where it's saying HTTP/1.1? HTTP/2 is working with zero-config on my Vercel site.
<img width="766" alt="CleanShot 2021-08-16 at 08 40 30@2x" src="https://user-images.githubusercontent.com/9113740/129572989-82e09497-1c15-4509-aeca-a42ab46d2eec.png">
